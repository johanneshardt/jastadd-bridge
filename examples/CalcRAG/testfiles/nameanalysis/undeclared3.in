let
	you = let
		me = 0.0
	in
		1.0
	end
in
	let
		us = me * you
	in
		us
	end
end
