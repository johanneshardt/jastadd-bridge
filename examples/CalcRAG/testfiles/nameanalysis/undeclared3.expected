Error at line 9: symbol 'me' is not declared
